src/components/Buttons/SortButton.js

```diff
@@ -0,0 +1,41 @@
+import React, { useState } from 'react';
+import styled from 'styled-components';
+import SortModal from '../Modals/SortModal';
+
+const SortButton = () => {
+  const [currSort, setCurrSort] = useState('응답률순');
+  const [isOpen, setIsOpen] = useState(false);
+
+  const handleButtonClick = () => {
+    setIsOpen(!isOpen);
```

Comment:
```suggestion
    setIsOpen(prev => !prev);
```
이전값을 활용해서 새로운 state를 만드는 경우에는 함수형 업데이트를 사용하는게 좀 더 안전합니다~!

Reply:
헉 그렇군요 새로운 사실 알고 갑니다~😱